nemo/collections/asr/parts/manifest.py

```diff
@@ -74,10 +76,33 @@ def item_iter(
 
                 yield item
 
+def __parse_open_stt(line: str, manifest_file: str) -> Dict[str, Any]:
+    manifest_path, _ = split(manifest_file)
+    line_list = line.split(",")
+    assert len(line_list)==3, f"Incorrect fields number: {len(line_list)}"
+    item = dict(
```

Comment:
Please update the this item dict with values from the manifest (if if exists) - at least for offset

Reply:
OpenStt dataset does not have offset values. There are only 3 comma-separated fields: audio_file, text_file, duration. https://github.com/snakers4/open_stt  I move this function out to a separate file.

Reply:
Yep I know it doesnt have offset values, but the audio processing pipeline is capable of slicing off segments using the two keys of offset and duration even from whole files. So we can augment the manifest for speech classification in the future with some modifications to the manifest. Anyway it can be handled in the future I suppose.